static/build/.tmp_update/script/network/update_networking.sh

```diff
@@ -330,6 +467,32 @@ echo_enabled() {
     fi
 }
 
+print_usage() {
+    echo "Usage: $0 {check|ftp|telnet|http|ssh|ntp|hotspot|disableall} {toggle|authed}"
+    exit 1
+}
+
+get_time() { # handles 2 types of NTP, instant from an API or longer from a time server, if the instant API check fails it will fallback to the longer ntp
+    response=$(curl -s http://worldtimeapi.org/api/ip.txt | grep -o 'utc_datetime: [^,]*' | cut -d ' ' -f 2)
+    if [ -z "$response" ]; then
+        log "NTP: Failed to get time from worldtimeapi.org, falling back to NTP."
+        ntpdate -u time.google.com 
+		touch /tmp/time_update
+		sync
+    else
+        utc_time=$(echo "$response" | cut -dT -f2 | cut -d+ -f1 | cut -d. -f1)
```

Comment:
```suggestion
        utc_time=$( echo "$response" | cut -d. -f1 | sed "s/T/ /")
```

Reply:
This will leave the time in, and replace the "T" with a space, so busybox date recognizes it

from `date -h`
```
Recognized TIME formats:
...
        YYYY-MM-DD hh:mm[:ss]
```

Reply:
Everything else worked fine on my tests.